dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases/ClickhouseDataSourceE2ETest.java

```diff
@@ -80,10 +82,10 @@ void testCreateClickhouseDataSource() {
 
         page.createDataSource(dataSourceType, dataSourceName, dataSourceDescription, ip, port, userName, pgPassword, database, jdbcParams);
 
-        new WebDriverWait(page.driver(), 10).until(ExpectedConditions.invisibilityOfElementLocated(
+        new WebDriverWait(page.driver(), Duration.ofSeconds(20)).until(ExpectedConditions.invisibilityOfElementLocated(

```

Comment:
we change value from 10 to 20 for longer timeout?

Reply:
> we change value from 10 to 20 for longer timeout?

Yes...